app/src/org/commcare/dalvik/activities/HomeButtons.java

```diff
@@ -166,6 +172,7 @@ public void update(HomeCardDisplayData cardDisplayData,
     private static View.OnClickListener getLogoutButtonListener(final CommCareHomeActivity activity) {
         return new View.OnClickListener() {
             public void onClick(View v) {
+                reportButtonClick(GoogleAnalyticsFields.LABEL_LOGOUT_BUTTON);
```

Comment:
Just that it seems verbose to add these in as code lines. In theory we could have this be a thing that happens automatically based on the button having an annotation or something if we reliably used a common onclicklistener. Now that the other stuff is less verbose I'm less concerned though